benchmark/single-source/AngryPhonebook.swift

```diff
@@ -15,26 +15,54 @@
 import TestsUtils
 import Foundation
 
-public let AngryPhonebook = BenchmarkInfo(
-  name: "AngryPhonebook",
-  runFunction: run_AngryPhonebook,
-  tags: [.validation, .api, .String],
-  legacyFactor: 7)
+let t: [BenchmarkCategory] = [.validation, .api, .String]
 
-let words = [
+public let AngryPhonebook = [
+  BenchmarkInfo(
```

Comment:
As I've said before:
> We'll leave the original `AngryPhonebook` benchmark without changes.

The reason is that we use these benchmarks for long term performance tracking between releases.

Please add the original `AngryPhonebook` back as it was. We can add the additional `AngryPhonebook.Latin` for direct comparison with the other additions. So I'd really go with `let latin = Array(words.prefix(20))` for short.

Reply:
Sorry, missed that.
I guess I need to return the `legacyFactor` too. It makes a huge difference.
And maybe not use the `setUpFunction` parameter to. What do you think? 
The previous implementation didn't have that.

Before
```
BenchmarkInfo(
  name: "AngryPhonebook",
  runFunction: run_AngryPhonebook,
  tags: [.validation, .api, .String],
  legacyFactor: 7)
```
After
```
BenchmarkInfo(
    name: "AngryPhonebook",
    runFunction: { angryPhonebook($0, words) },
    tags: t,
    legacyFactor: 7),
```

Is that good?

Reply:
I suspect replacing the old `run_AngryPhonebook` with the shared `angryPhonebook` will be a measurable difference due to the use of `blackHole`… I'll kick the benchmarks and we'll see.

Reply:
No measurable change. I think we are good to go as is.